Tests/CreateAPITests/GenerateOptionsTests.swift

```diff
@@ -605,15 +605,15 @@ final class GenerateOptionsTests: GenerateBaseTests {
         try compare(package: "edgecases-int32-int64")
     }
     
-    func testEdgecasesGenerateCodingKeys() throws {
+    func testEdgecasesGenerateEnumCodingKeys() throws {
         // GIVEN
         let command = try Generate.parse([
             pathForSpec(named: "edgecases", ext: "yaml"),
             "--output", temp.url.path,
             "--package", "edgecases-coding-keys",
             "--config", config("""
             entities:
-                isGeneratingCustomCodingKeys: false
+                isUsingStringsForCodingKeys: false
```

Comment:
Since this test was previously the only one to check the `StringCodingKey` usage, I've now flipped that so that it tests the enum `CodingKey` usage